src/main/java/reactor/netty/DisposableServer.java

```diff
@@ -17,33 +17,55 @@
 package reactor.netty;
 
 import java.net.InetSocketAddress;
+import java.net.SocketAddress;
 
 /**
  * Holds contextual information for the underlying server
  *
  * @author Stephane Maldini
+ * @author Violeta Georgieva
  * @since 0.8
  */
 public interface DisposableServer extends DisposableChannel {
 
+	@Override
+	default SocketAddress address() {
+		return channel().localAddress();
+	}
+
 	/**
 	 * Returns the server's host string. That is, the hostname or in case the server was bound
 	 * to a literal IP address, the IP string representation (rather than performing a reverse-DNS
 	 * lookup).
 	 *
 	 * @return the host string, without reverse DNS lookup
+	 * @throws UnsupportedOperationException when Unix Domain Sockets
 	 * @see DisposableChannel#address()
 	 * @see InetSocketAddress#getHostString()
 	 */
 	default String host() {
-		return address().getHostString();
+		return "";
+	}
+
+	/**
+	 * Returns the server's path to the domain socket.
+	 * Returns {@code null} in case it is not Unix Domain Sockets.
+	 *
+	 * @return the path to the domain socket
+	 * @throws UnsupportedOperationException when it is not Unix Domain Sockets
```

Comment:
the default implementation returns `""` while the javadoc says it throws (I prefer the former)

Reply:
done